gps/models.py

```diff
@@ -0,0 +1,19 @@
+"""Define the models to store gps data"""
+
+from django.db import models
+from covidnas.models import User
```

Comment:
Importing of user model should be done by referencing the AUTH_USER_MODEL of settings.py file [docs](https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#referencing-the-user-model)

Reply:
oh yeah. that is a mistake. I will remedy that.